src/shogun/features/DotFeatures.cpp

```diff
@@ -358,19 +365,19 @@ void CDotFeatures::benchmark_dense_dot_range(int32_t repeats)
 			dense_dot_range(out, 0, num, alphas, w, d, 23);
 
 #ifdef DEBUG_DOTFEATURES
-    CMath::display_vector(out, 40, "dense_dot_range");
```

Comment:
I just converted all the spaces to tabs since that is what is mentioned in the coding guide.
It must have been spaces there before.